apiary.apib

```diff
@@ -3049,6 +3049,18 @@ Get a list of tracked time.
                             + ticket
                     + id: `58f5b799-51c4-4eb9-8308-b1aa02e0a873` (string)
                 + invoiceable: true (boolean)
+                + relates_to (array)
```

Comment:
My eye still slightly twitches reading this. Should this be similar to the link(s) we had on events? Just not sure if this is the right "term" in a time tracking context. But could also just be me.

Reply:
Links on calendar events are updateable explicitly. In this case they are calculated (on retrieval or saving, that needs to be decided) from the subject of the time tracking. So it's not exactly the same.

Reply:
`tracked_on`?